tests/testsuite/run.rs

```diff
@@ -1416,6 +1416,23 @@ fn default_run_workspace() {
     p.cargo("run").with_stdout("run-a").run();
 }
 
+#[cargo_test]
+fn print_env_verbose() {
+    let p = project()
+        .file(
+            "Cargo.toml",
+            r#"
+                [package]
+                name = "a"
+                version = "0.0.1"
+            "#,
+        )
+        .file("src/main.rs", r#"fn main() {println!("run-a");}"#)
+        .build();
+
+    p.cargo("run -vv").with_stderr("LD_LIBRARY_PATH").run();
```

Comment:
This contains two shortcomings:

* [Different platforms have different library paths](https://doc.rust-lang.org/nightly/cargo/reference/environment-variables.html#dynamic-library-paths). This won't pass on Windows and macOS.
* This matches both `rustc` and the final binary invocations. We can't tell whether the env is from which.

What about this? `[..]` for wildcard matches, and we match one env vars to verify the behavior.

```suggestion
    p.cargo("run -vv")
        .with_stderr(
            "\
[COMPILING] a v0.0.1 ([CWD])
[RUNNING] `[..]CARGO_MANIFEST_DIR=[CWD][..] rustc --crate-name a[..]`
[FINISHED] dev [unoptimized + debuginfo] target(s) in [..]
[RUNNING] `[..]CARGO_MANIFEST_DIR=[CWD][..] target/debug/a[EXE]`",
        )
        .run();
```